### PR TITLE
#3605 refactor reindex script and add to repair menu

### LINKED ISSUE #3605: Add reindex Bitcoin chainstate to the Repair menu
There is also delete transaction index in the repair menu - but not the "reindex chainstate" like offered by the `/config-scripts/network.reindex.sh` script. Add this and make the progress info as part of the background task.

Improve that script, add status/progress info and make it part of the repair menu.

Cross reference: #3016 & https://github.com/rootzoll/raspiblitz/issues/3178#issuecomment-1381166928

### PR DESCRIPTION
**Pull request by rootzoll**
fixes #3605

### CODE REVIEW

File: home.admin/config.scripts/blitz.subscriptions.py [hunk 1]
```diff
@@ -225,7 +225,7 @@ def main():
 
     choices = list()
     choices.append(("LIST", "My Subscriptions"))
-    #choices.append(("NEW1", "+ IP2TOR Bridge (paid)"))
+    choices.append(("NEW1", "+ IP2TOR Bridge (paid)"))
```
Comment (openoms) [inline home.admin/config.scripts/blitz.subscriptions.py]: great, will IP2Tor be back in action?

File: home.admin/config.scripts/network.reindex.sh [hunk 2]
```diff
@@ -1,93 +1,53 @@
 #!/bin/bash
 
 # command info
-if [ "$1" = "-h" ] || [ "$1" = "-help" ]; then
- echo "script to run re-index if the blockchain (in case of repair)"
- echo "run to start or monitor re-index progress"
+if [ $# -eq 0 ] || [ "$1" = "-h" ] || [ "$1" = "-help" ]; then
+ echo "script to run re-index if the blockchain - blocks will not be deleted but re-indexed"
+ echo "will trigger reboot after started and progress can be monitored thru normal sync status"
+ echo "network.reindex.sh reindex [main|test|sig] --> use to start re-index chain"
```
Comment (openoms) [inline home.admin/config.scripts/network.reindex.sh]: just a nit that in most scripts we use the full chain name like mainnet | testnet | signet

File: home.admin/config.scripts/blitz.subscriptions.py (hunk 1 above)
Comment (rootzoll) [reply home.admin/config.scripts/blitz.subscriptions.py]: Thas something I experimented with - slipped into this PR - will remove here.

File: home.admin/config.scripts/network.reindex.sh (hunk 2 above)
Comment (rootzoll) [reply home.admin/config.scripts/network.reindex.sh]: I can adjust

**Review by openoms — APPROVED** [review]

**Pull request merged**